packages/material-ui/src/Tooltip/Tooltip.js

```diff
@@ -21,20 +20,22 @@ export const styles = theme => ({
   },
   /* Styles applied to the tooltip (label wrapper) element. */
   tooltip: {
-    backgroundColor: theme.palette.grey[700],
+    backgroundColor: 'rgba(97, 97, 97, 0.9)',
```

Comment:
Do we have a utility function that takes an rgb value and applies opacity?
```suggestion
    backgroundColor: insertNameOfUtilityFunction(theme.palette.grey[700], 0.9),
```

Reply:
Didn't find one, but I wondered that, too

Reply:
`lighten()` 

I'd like to rename that when we publicly expose the colorManipulator API. It's confusing.

Reply:
Okay, I didn't search for "lighten" to make a color less opaque. :sweat_smile: What about `setAlpha`?

Reply:
@eps1lon Done

Reply:
Sorry, my mistake. I should have put "IIRC". (Or I should have double checked). `setOpacity` or `setAlpha` might be a better name than `fade`. `fade` sounds like it reduces saturation.